scripts/render-rule-coverage-gif.py

```
@@ -318,18 +337,20 @@ def main():
                             observed=csv_set(args.observed))
 
-    n_verified = sum(1 for r in order if state[r] == "verified")
+    lit_seq = [(r, *by_rule.get(r, ("(fires without being asserted)", "")))
+               for r in order if state[r] == "verified"]
+    n_verified = len(lit_seq)
     frames = []
     for lit in range(n_verified + 1):
         frames.append(draw(order, state, lit, args.title, args.agent, labels,
                            args.note, probe_attacks, csv_set(args.exclude),
-                           (args.width, args.height)))
+                           (args.width, args.height), lit_seq))
```

Ruff (0.16.0)

[warning] Use a list comprehension to create a transformed list

(PERF401)